aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -423,6 +424,39 @@ impl AptosVMImpl {
             .or_else(|err| convert_prologue_error(transaction_validation, err, log_context))
     }
 
+    /// Run the prologue for a multisig transaction. This needs to
+    pub(crate) fn run_multisig_prologue<S: MoveResolverExt>(
```

Comment:
I wonder if we really benefit much from running the prologue in Move.... versus the cost of doing so... we should try a forge run with a disabled prologue.....

@zekun000 / @igor-aptos

Reply:
I think the general rationale here is that Move code is closer to the data model (e.g. account, multisig account) and thus is more natural and easier to maintain than having Rust touch the data structures defined in Move.